db/active_replicator.go

```diff
@@ -208,7 +208,7 @@ func connect(arc *activeReplicatorCommon, idSuffix string) (blipSender *blip.Sen
 	arc.replicationStats.NumConnectAttempts.Add(1)
 
 	var originPatterns []string // no origin headers for ISGR
-	blipContext, err := NewSGBlipContext(arc.ctx, arc.config.ID+idSuffix, originPatterns)
```

Comment:
As discussed offline, we are not going to pass in a cancellation context for a passive ISGR peer at this time to limit the scope of this PR.

I think it would be safe to add this here, but it's not needed since `DatabaseContext.Close` will call `DatabaseContext.sgReplicateMgr.Stop` which stops all replicators. If you agree with this, I think adding a comment to this code would be helpful, because I'll ask this question again.